litmus-portal/frontend/src/views/ChaosWorkflows/BrowseSchedule/styles.ts

```diff
@@ -43,7 +42,7 @@ const useStyles = makeStyles((theme) => ({
   },
   tableMain: {
     marginTop: theme.spacing(4.25),
-    border: `1px solid ${theme.palette.border.main}`,
+    border: '1px solid rgba(0, 0, 0, 0.05)',
```

Comment:
can we use disabled background here?

Reply:
Sure, making the changes.
Thanks @ishangupta-ds

Reply:
According to figma tables aren't using borders, the internal gird is having border main